spec/plans/discover.fmf

```diff
@@ -105,108 +91,9 @@ description: |
     summary: Discover available tests using the fmf format
     description: |
         Use the `Flexible Metadata Format`_ to explore all
-        available tests in given repository. The following
-        parameters are supported:
-
-        url
-            Git repository containing the metadata tree.
-            Current git repository used by default.
-        ref
-            Branch, tag or commit specifying the desired git
-            revision. Defaults to the remote repository's default
-            branch if url given or to the current ``HEAD`` if url
-            not provided.
-
-            Additionally, one can set ``ref`` dynamically. This is
-            possible using a special file in tmt format stored in
-            the *default* branch of a tests repository. This special
-            file should contain rules assigning attribute ``ref``
-            in an `adjust` block, for example depending on a test
-            run context.
-
-            Dynamic ``ref`` assignment is enabled whenever a test
-            plan reference has the format ``ref: @FILEPATH``.
-        path
-            Path to the metadata tree root. Must be relative to
-            the git repository root if url provided, absolute
-            local filesystem path otherwise. By default ``.`` is
-            used.
-
-        See also the `fmf identifier`_ documentation for details.
-        Use the following keys to limit the test discovery by test
-        name, an advanced filter or link:
-
-        test
-            List of test names or regular expressions used to
-            select tests by name. Duplicate test names are allowed
-            to enable repetitive test execution, preserving the
-            listed test order.
-        link
-            Select tests using the :ref:`/spec/core/link` keys.
-            Values must be in the form of ``RELATION:TARGET``,
-            tests containing at least one of them are selected.
-            Regular expressions are supported for both relation
-            and target. Relation part can be omitted to match all
-            relations.
-        filter
-            Apply advanced filter based on test metadata
-            attributes. See ``pydoc fmf.filter`` for more info.
-        exclude
-            Exclude tests which match a regular expression.
-        prune
-            Copy only immediate directories of executed tests and
-            their required files.
```

Comment:
This section described the group of all options which can be used for selecting tests. On the new page they are scattered across all options. Would it make sense to include a small section in the docstring listing these all with a simple example. Users then will be able to look them up in the long list of options for details but giving them an overview of what can be used for filtering would be beneficial I'd say.

Reply:
@psss Added a list of the supported keys for limiting test discovery

Reply:
Thanks.